### DISCUSSION TITLE
How can I hide icons on the desktop?

### DISCUSSION THREAD

**Discussion by Barugon** [pop-os/cosmic-epoch#1089] [Q&A] [answered]
If I put files or folders in the desktop folder, how can I have them not show up on the desktop (like GNOME)?

**Comment by jacobgkau** [comment, marked as answer]
Right-click the desktop, click `Desktop view options...` and turn off the `Desktop folder content` option that's on by default.

![screenshot-2024-10-09-14-54-06](https://github.com/user-attachments/assets/9b170ac9-7c36-4333-a489-0c600bc715f6)

**Reply by Barugon** [reply]
Thanks.
